easybuild/easyconfigs/m/MapSplice/MapSplice-2.2.1-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'MapSplice'
+version = '2.2.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.netlab.uky.edu/p/bioinfo/%(name)s2'
+description = """MapSplice is a software for mapping RNA-seq data to reference genome for splice
+ junction discovery that depends only on reference genome, and not on any further annotations."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://protocols.netlab.uky.edu/~zeng']
+sources = ['%(name)s-v%(version)s.zip']
+
+patches = [
+    '%(name)s-%(version)s_fix-build-gcc.patch',
+]
```

Comment:
style nitpicking: collapse this to a single line, drop empty line before & after

```python
sources = ['%(name)s-v%(version)s.zip']
patches = ['%(name)s-%(version)s_fix-build-gcc.patch']
checksums = [
  ...
```

Reply:
@boegel I changed this.